packages/composites/linear-finance/src/endpoint/allocations.ts

```diff
@@ -2,7 +2,21 @@ import { AdapterInputError, Requester, Validator } from '@chainlink/ea-bootstrap
 import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
 import { Decimal } from 'decimal.js'
 import moment from 'moment-timezone'
-import { XBCI, XLCI } from '../config'
+
+const endpointConfig: { [index: string]: { url: string; symbolKey: 'symbol' | 'slug' } } = {
+  xlci: {
+    url: '/v1/index/xangle-largecap', //TODO remove support? https://pro-api.xangle.io/#section/Version-History
```

Comment:
Do you want to leave these comments here?

Reply:
Removed